io/json/include/detray/io/json_utilities_io.hpp

```diff
@@ -0,0 +1,142 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
Should the grids really go into utilities, because in detray they are not kept in the utilities dir ? Btw, do all detectors in ACTS use grids in the geometry?

Reply:
No hard feelings there, I will synchronise then.

Reply:
As for the second question: we will only support those with indexing and grids for Detray writing, I suppose.